tests/networks.spec.ts

```diff
@@ -4,148 +4,378 @@ import {
-  await activateOverride(page, "DNS search");
-  await page.getByLabel("DNS search").fill("abc");
-
-  await page.getByText("IPv4").click();
-  await activateOverride(page, "IPv4 DHCP expiry");
-  await page.getByLabel("IPv4 DHCP expiry").fill("2h");
-
-  await page.getByText("IPv6").click();
-  await activateOverride(page, "IPv6 DHCP expiry");
-  await page.getByLabel("IPv6 DHCP expiry").fill("3h");
-  await activateOverride(page, "IPv6 DHCP stateful");
-  await setOption(page, "IPv6 DHCP stateful", "true");
-
-  await saveNetwork(page, network);
-
-  await visitNetwork(page, network);
-  await page.getByRole("cell", { name: "A-new-description" }).click();
-
-  await page.getByTestId("tab-link-Configuration").click();
-  await assertReadMode(page, "IPv4 NAT", "false");
-  await assertReadMode(page, "IPv6 NAT", "false");
-
-  await page
-    .getByLabel("Network form navigation")
-    .getByText("Bridge", { exact: true })
-    .click();
-  await assertReadMode(page, "MTU", "1300");
-  await assertReadMode(page, "Bridge driver", "native");
-
-  await page.getByText("DNS", { exact: true }).click();
-  await assertReadMode(page, "DNS domain", "abc");
-  await assertReadMode(page, "DNS mode", "none");
-  await assertReadMode(page, "DNS search", "abc");
-
-  await page.getByText("IPv4", { exact: true }).click();
-  await assertReadMode(page, "IPv4 DHCP expiry", "2h");
-
-  await page.getByText("IPv6", { exact: true }).click();
-  await assertReadMode(page, "IPv6 DHCP expiry", "3h");
-  await assertReadMode(page, "IPv6 DHCP stateful", "true");
+test.describe("physical type", () => {
+  test.beforeAll(async ({ browser, browserName }) => {
+    // network names can only be 15 characters long
+    network = `${browserName.substring(0, 2)}-${network}`;
+    const page = await browser.newPage();
+    await createNetwork(page, network, "physical");
+    await page.close();
+  });
+
+  test.afterAll(async ({ browser }) => {
+    const page = await browser.newPage();
+    await deleteNetwork(page, network);
+    await page.close();
+  });
+
+  test("configure physical network DNS nameservers", async ({ page }) => {
+    const DNS_NAMESERVERS = "32.152.61.210";
+
+    await prepareNetworkTabEdit(page, "DNS", network);
+
+    await setTextarea(page, "DNS nameservers", DNS_NAMESERVERS);
+  });
+
+  test("set physical network IPv4 routes and gateway", async ({ page }) => {
+    const options = ["true", "false"];
+    const OVN_RANGES = "10.141.1.100-10.141.1.110";
+    const IPV4_ROUTES = "10.141.1.1";
+
+    await prepareNetworkTabEdit(page, "IPv4", network);
+
+    await setTextarea(page, "IPv4 OVN ranges", OVN_RANGES);
+    await setTextarea(page, "IPv4 gateway", NETWORK_CONSTANTS.IPV4_ADDRESS);
+    await activateOverride(page, "IPv4 routes Additional IPv4");
+    await page.getByLabel("IPv4 routes").fill(IPV4_ROUTES);
+    for (const option of options) {
+      await setOption(page, "IPv4 routes anycast", option);
+    }
```

Comment:
I am not a fan of repeating the for loop in every test case. Maybe add a helper `checkAllOptions(page, "name", options)` that we can use in all cases instead. The helper can have the for loop internally, so we have it exactly once and don't need to repeat it.

Reply:
Thanks for adjusting this :+1: